frontend/src/components/ViewTrips/edit-trip-button.js

```diff
@@ -4,16 +4,15 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import Button from 'react-bootstrap/Button';
 
 /**
- * {@link RawTripData} defined originally in `ViewTrips/save-trip-modal.js`.
+ * {@link TripData} defined originally in `ViewTrips/trip.js`.
  */
```

Comment:
I think I commented this in a previous review but do you need this link in the jsdocs?

Reply:
Yes you did mention this before.

To reiterate, I have been putting links across all my files where I use a typedef that wasn't defined in the file for reference (if they want to find that type definition in the editor).